### DISCUSSION TITLE
Error 404 trying to find chart.js

### DISCUSSION THREAD

**Discussion by ThomasHingray** [community/community#111186] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi everybody,

I couldn't find the answer anywhere (or I'm not yet very good at searching, I don't know)...
I've tried to upload and host my work on gh Pages. At first, only html could be read and CSS and Javascript were missing. I've tried to use relative path and now almost everything works... Except the use of chart.js, the background and the "Rejouer" (Play again) button in the end... May I have some hints?

GH page: https://thomashingray.github.io/testDino/
Repository: https://github.com/ThomasHingray/testDino

PS: I don't know which of my english or my code is the worst at the moment but be sure that I'm working on both and grateful for advices ;)

**Comment by queenofcorgis** [comment]
Hi there, 

Thanks for sharing the project - I had fun taking the quiz! It seems like everything is working for me though... could you confirm if this is resolved or share more on what isn't working?

**Reply by ThomasHingray** [reply, marked as answer]
Hi ! 

I'm glad you liked it ! I managed to find the answer ! I commited with node_module folder and gh page didn't manage to find chart.js in it. So I had to take it out and delete node folder and now it works!

**Reply by queenofcorgis** [reply]
Amazing job (I'm a stegosaurus) - thanks for letting us know the answer!
